repos/spack_repo/builtin/packages/moab/package.py

```diff
@@ -55,6 +55,7 @@ class Moab(AutotoolsPackage):
     variant("fbigeom", default=False, description="Enable FBiGeom interface")
     variant("coupler", default=False, description="Enable mbcoupler tool")
     variant("dagmc", default=False, description="Enable DagMC tool")
+    variant("tempest", default=False, description="Enable mbtempest tool")
```

Comment:
And you can add a conditional variant:
```suggestion
    variant("tempest", default=False, when="@5.1: +hdf5 +netcdf", description="Enable mbtempest tool")
```
(note that the suggestion probably needs refactoring for style related issues)

Reply:
Great suggestion!